spec/stack_master/parameter_resolvers/stack_output_spec.rb

```diff
@@ -37,6 +37,7 @@ def resolve(value)
     let(:outputs) { [] }
 
     before do
+      allow(StackMaster.cloud_formation_driver).to receive(:region).and_return(region)
```

Comment:
Stub out the default AWS region. This prevents specs failing in development environments where the default
region is not `us-east-1`.